src/main/java/com/example/animalshelter/model/AnimalGender.java

```diff
@@ -0,0 +1,8 @@
+package com.example.animalshelter.model;
+
+public enum AnimalGender {
+
+    UNKNOWN,
```

Comment:
I think the UNKNOW value is not necessary ;)

Reply:
yes, this was an option when an user don't want to provide all details initially(like a default value). I will remove it.